packages/account/src/Components/poi/status/not-required/__test__/onfido-not-required.spec.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
```

Comment:
can we rename the spec filename to not-required.spec.js .
Also the folder name should be `__tests__`.